example/takeoff_land/CMakeLists.txt

```diff
@@ -9,18 +9,19 @@ set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
 
 # This finds thread libs on Linux, Mac, and Windows.
 find_package(Threads REQUIRED)
+message(${CMAKE_SOURCE_DIR})
 
 include_directories(
     # Not needed if installed system-wide
-    ../install/include
+    ${CMAKE_SOURCE_DIR}/../../install/include
```

Comment:
:) ${CMAKE_SOURCE_DIR} - refers to source directory where CMakeLists.txt resides i.e, `/home/intel/DroneCore/example/takeoff_land`. Hence, going back twice.